WordPress/src/main/java/org/wordpress/android/ui/main/ActionListItemViewHolder.kt

```diff
@@ -0,0 +1,25 @@
+package org.wordpress.android.ui.main
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import android.widget.ImageView
+import android.widget.TextView
+import androidx.recyclerview.widget.RecyclerView.ViewHolder
+import org.wordpress.android.R
+import org.wordpress.android.ui.main.MainActionListItem.CreateAction
+
+open class ActionListItemViewHolder(
```

Comment:
I don't think this should be `open`

Reply:
I was keeping it open in case we want to have more View Holder in the future, but I agree that currently it's better to remove since we are not subclassing it. Changed here 319e7289aa7f7c351b288adb56cf0efb36b496a1